### ISSUE TITLE
What's the correspondence of joints in Posetrack format to those in COCO format?

### ISSUE THREAD

**Issue by Odaimoko** [Microsoft/human-pose-estimation.pytorch#43]
In your paper, you used pretrained model on COCO and then fine-tuned on Posetrack. But their points format are different, so there must be a conversion otherwise the model could not be used. How did you convert COCO format to Posetrack 17 in your experiment?

**Comment by nikhilaravi** [comment]
I am also interested in evaluating the model on posetrack data.

**Comment by sklf** [comment]
> > I am also interested in evaluating the model on posetrack data.
> 
> @nikhilaravi
> 
> Actually I removed the last layer of author-provided COCO model. I added random initialized layer and the output size = 15. I trained this on posetrack 17 training set.

Hello，what’ s your model peformance? And can you share your code about your operation mentioned above? Thanks!

**Comment by Odaimoko** [comment]
> > > I am also interested in evaluating the model on posetrack data.
> > 
> > 
> > @nikhilaravi
> > Actually I removed the last layer of author-provided COCO model. I added random initialized layer and the output size = 15. I trained this on posetrack 17 training set.
> 
> Hello，what’ s your model peformance? And can you share your code about your operation mentioned above? Thanks!

Sure. Reply to me if you still want it and I will add some comments. It reached 69.6 mAP on Posetrack 2017 validation set.

**Comment by zhuzhu18** [comment]
Also I am interested in the code about tracking, can you share me about the tracking code. Much thanks :yum:

**Comment by Odaimoko** [comment]
> > > > > I am also interested in evaluating the model on posetrack data.
> > > > 
> > > > 
> > > > @nikhilaravi
> > > > Actually I removed the last layer of author-provided COCO model. I added random initialized layer and the output size = 15. I trained this on posetrack 17 training set.
> > > 
> > > 
> > > Hello，what’ s your model peformance? And can you share your code about your operation mentioned above? Thanks!
> > 
> > 
> > Sure. Reply to me if you still want it and I will add some comments. It reached 69.6 mAP on Posetrack 2017 validation set.
> 
> Also I am interested in the code about tracking, can you share me about the tracking code. Much thanks 😋
> My email can be reached at [zhuzhihua18@mails.ucas.edu.cn](mailto:zhuzhihua18@mails.ucas.edu.cn)

You can browse my repo. Its name is nise_embedding and it is a mess...you may have to look into it a bit and feel free to propose issues.
